README.md

```diff
@@ -147,6 +147,82 @@ This project was made as Go learning opportunity. The initial release was create
 	@reboot gitlab-ci-multi-runner run &>gitlab-ci-multi-runner.log
 	```
 
+### Docker image installation and configuration (run gitlab-ci-multi-runner as a container)
+
+1. Pull the image (optional):
+  ```bash
+  $ docker pull ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+1. Start the container:
+  ```bash
+  $ docker run -d --name multi-runner --restart always \
+      ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+  If you are planning on using Docker as the method of spawing runners you'll need to mount your docker socket like so:
+  ```bash
+  $ docker run -d --name multi-runner --restart always \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+  Additionally, you can mount your own custom data folder to be used for loading the config and any other resources:
```

Comment:
I think that it should be obligatory. We first create data container and pass that data container with --volumes-from or with --volumes as you described later.